src/config.ts

```diff
@@ -0,0 +1,84 @@
+import {cosmiconfigSync} from 'cosmiconfig';
+import {Validator} from 'jsonschema';
+import {merge} from 'lodash';
+
+export interface SnowpackConfig {
+  options: {
+    babel?: boolean;
+    clean?: boolean;
+    dest?: string;
+    exclude?: string[];
+    externalPackage?: string[];
+    include?: string[];
+    nomodule?: string;
+    nomoduleOutput?: string;
+    noSourceMap?: boolean;
```

Comment:
Sidenote: in the help text, the flag is `--no-source-map`. But in the `yargs`, it’s `sourceMap`. A little confused if this is a positive or negative flag?